Source/Charts/Data/Implementations/Standard/BarChartDataSet.swift

```diff
@@ -149,6 +156,26 @@ open class BarChartDataSet: BarLineScatterCandleBubbleChartDataSet, BarChartData
     /// the alpha value (transparency) that is used for drawing the highlight indicator bar. min = 0.0 (fully transparent), max = 1.0 (fully opaque)
     open var highlightAlpha = CGFloat(120.0 / 255.0)
     
+    /// array of gradient colors [[color1, color2], [color3, color4]]
+    open var barGradientColors: [[NSUIColor]]?
```

Comment:
Why is this a nested array?

Reply:
Nested array is array of colors for each bar.